### PR TITLE
Adding login page and add tests

### LINKED ISSUE #78: Adding login page

### PR DESCRIPTION
**Pull request by idosinger98**
The changes that were made:

1. Add login page

2. Add tests to the login page

3. Update landing/base.html and delete  templates/base.html
Resolve #78 
image

### CODE REVIEW

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by Yarboa — COMMENTED** [review]
Not working for me,

[23/May/2023 12:30:23] "GET /accounts/login/ HTTP/1.1" 404 5335
Not Found: /login
[23/May/2023 12:30:42] "GET /login HTTP/1.1" 404 5296
Not Found: /accounts/login/
[23/May/2023 12:31:14] "GET /accounts/login/?next=/ HTTP/1.1" 404 5342
Not Found: /accounts/login

**Comment by Yarboa** [comment]
> > Not working for me,
> > [23/May/2023 12:30:23] "GET /accounts/login/ HTTP/1.1" 404 5335 Not Found: /login [23/May/2023 12:30:42] "GET /login HTTP/1.1" 404 5296 Not Found: /accounts/login/ [23/May/2023 12:31:14] "GET /accounts/login/?next=/ HTTP/1.1" 404 5342 Not Found: /accounts/login
> 
> I think you entered the wrong url image

It is failing also in tests
Please refer the 2nd get /login returned 404

**Comment by idosinger98** [comment]
> > > Not working for me,
> > > [23/May/2023 12:30:23] "GET /accounts/login/ HTTP/1.1" 404 5335 Not Found: /login [23/May/2023 12:30:42] "GET /login HTTP/1.1" 404 5296 Not Found: /accounts/login/ [23/May/2023 12:31:14] "GET /accounts/login/?next=/ HTTP/1.1" 404 5342 Not Found: /accounts/login
> > 
> > 
> > I think you entered the wrong url image
> 
> It is failing also in tests Please refer the 2nd get /login returned 404

Done.

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by idoyek — APPROVED** [review]
LGTM

**Review by Yarboa — COMMENTED** [review]
Please add Login Form validation

File: account/tests/test_login.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+USERNAME = "testusername3"
+PASSWORD = 'testpassword3'
+
+
+@pytest.mark.django_db
+class TestLoginView:
+    def test_enter_login_page(self, client):
+        response = client.get('/login/')
+        assert response.status_code == 200
```
[the remark this answers is not in the document]
Comment (idosinger98) [reply account/tests/test_login.py:14]: Done @Yarboa .

**Comment by idosinger98** [comment]
> Please add Login Form validation

Done @Yarboa .

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**